os/mm/mm_heap/mm_check_heap_corruption.c

```diff
@@ -56,6 +56,7 @@
 #include <tinyara/config.h>
 #include <tinyara/arch.h>
 #include <tinyara/mm/mm.h>
+#include <sys/prctl.h>
```

Comment:
Please add condition, `TASK_NAME_SIZE > 0` and `DEBUG_MM_HEAPINFO`.

Reply:
Done